Formula/benerator.rb

```diff
@@ -0,0 +1,97 @@
+class Benerator < Formula
+  desc "Tool for realistic test data generation"
+  homepage "https://rapiddweller.github.io/homebrew-benerator/"
+  url "https://github.com/rapiddweller/rapiddweller-benerator-ce/releases/download/3.1.0/rapiddweller-benerator-ce-3.1.0-jdk-11-dist.tar.gz"
+  sha256 "194feb051ae18cfcd407b8e1668ce9c60561394bc454f9fc9747c274166843bc"
+
+  license "Apache-2.0"
+  depends_on "openjdk"
+
+  def install
+    # Remove unnecessary files
+    rm_f Dir["bin/*.bat", "bin/pom.xml"]
+
+    # Installs only the "bin" and "lib" directories from the tarball
+    libexec.install Dir["bin", "lib"]
+    # Generate a script that sets the necessary environment variables
+    (bin/"benerator").write_env_script(
+      libexec / "bin/benerator",
+      JAVA_HOME:      Language::Java.java_home,
```

Comment:
```suggestion
      JAVA_HOME:      Language::Java.overridable_java_home_env,
```

Reply:
> 

This doesn't set the JAVA_HOME properly...
Can this be something like : 
```suggestion
         JAVA_HOME:      ENV["JAVA_HOME"] || Language::Java.java_home ,
```

Reply:
Update on e46f3f3135d70263769cebb65051223c65039582